apps/aehttp/priv/swagger.json

```diff
@@ -1453,7 +1453,7 @@
         }
       }
     },
-    "/account/balance/{account_pubkey}" : {
+    "/account/{account_pubkey}/balance" : {
```

Comment:
❤️

Reply:
Hard to read the URL in a small browser window, I guess.
(In manual testing I sometimes change balance in txs and with my small url bar it is nice that it is at the beginning.)

But never mind that use-case